scripts/table_updates/update_data_table.py

```diff
@@ -706,9 +706,12 @@ def custom_fix_for_cpt_seq_data(
         config: dict,
         cohort: str = "",
         comment: str = "",
+        replace_patient_tier1a: bool = False, 
+        replace_sample_tier1a: bool = False, 
     ) -> None:
-    # unlist form column in master table
-    master_table["form"] = master_table["form"].apply(lambda x: ", ".join(x))
+    # unlist form column in master table only when not replacing tier1a variables
```

Comment:
Why is this needed? Can you put this in the comment description

Reply:
Because the form column [has already been reformatted](https://github.com/Sage-Bionetworks/genie-bpc-pipeline/blob/b30af62ff477306f58975db3d53501f35674ba86/scripts/table_updates/update_data_table.py#L643) if we turn on replace_patient_tier1a or replace_sample_tier1a. And we do need to unlist this column if none of the tier1a toggle is on